src/gui/notificationwidget.cpp

```diff
@@ -33,8 +34,8 @@ void NotificationWidget::setActivity(const Activity& activity)
 {
     _myActivity = activity;
 
-    Q_ASSERT( !activity._accName.isEmpty() );
     _accountName = activity._accName;
+    RASSERT(!_accountName.isEmpty(), _accountName = "<missing account name>");
```

Comment:
This would be a programming error and wouldn't result in data corruption. I'd keep the Q_ASSERT.